README.md

```diff
@@ -60,6 +60,26 @@ If someone already merged it but didn't release it, then it can be released:
 $ ../../bin/release-module
 ```
 
+### Stable releases
+
+All the release scripts assume they're releasing to master. For a stable release, additional parameters are needed. All of the above scripts accept parameters. The first is the directory where to execute.
+
+To create a release PR for the current directory to the 11.1-stable branch, first get your working directory set up correctly. This means that git should point to the correct code and `CHANGELOG.md` is prepared. Bundler should also have all gems installs. In the regular workflow, the `changelogs` command does this, but [Github Changelog Generator](https://github.com/github-changelog-generator/github-changelog-generator) doesn't really support non-linear releases (with branches) so in practice it's easier to do so by hand.
```

Comment:
```suggestion
To create a release PR for the current directory to the 11.1-stable branch, first get your working directory set up correctly. This means that git should point to the correct code and `CHANGELOG.md` is prepared. Bundler should also have all gems installed. In the regular workflow, the `changelogs` command does this, but [Github Changelog Generator](https://github.com/github-changelog-generator/github-changelog-generator) doesn't really support non-linear releases (with branches) so in practice it's easier to do so by hand.
```

Reply:
If I read this paragraph I still find I am not sure *what* to do with respect to "git should point to the correct code" and "CHANGELOG.md is prepared". I think this is complicated by the previous release sections that talk about generating a changelog. Does this translate to:

 1) Create a release branch based on the X.Y version you are releasing, e.g. 11.1-stable
 2) Cherry pick changes into the release branch via PRs
 3) Open a PR that bumps the version, and modifies the CHANGELOG. This is done manually by the releaser.

Reply:
Well, that's what you get when you write documentation at the end of a Friday afternoon ;)